CHANGELOG.md

```diff
@@ -1,14 +1,20 @@
-## 4.0.2 (Unreleased)
+## Next (Unreleased)
 
 N/A
 
-## 4.0.1
+## 4.0.2
```

Comment:
should this be 0.4.1 ?

Reply:
dang it good catch, thanks Josh will fix in the Go bump PR